patches/circom_runtime+0.1.24.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/node_modules/circom_runtime/build/main.cjs b/node_modules/circom_runtime/build/main.cjs
+index 235f1b6..3098349 100644
+--- a/node_modules/circom_runtime/build/main.cjs
++++ b/node_modules/circom_runtime/build/main.cjs
+@@ -77,7 +77,7 @@ async function builder(code, options) {
+ 
+     options = options || {};
+ 
+-    let memorySize = 32767;
++    let memorySize = 0;
```

Comment:
how sure are we that 0 works and it just dynamically allocates correctly + efficiently?

testing seems to work but do you know if the underlying library actually makes the memory grow dynamically

Reply:
If it doesn't work we can always revert. I am planning on doing more research on this in order to be able to confidently submit a PR to the actual repo. IMO based on the testing I've done this is pretty safe.